src/coreclr/vm/threadsuspend.cpp

```diff
@@ -3050,7 +3050,6 @@ BOOL Thread::RedirectCurrentThreadAtHandledJITCase(PFN_REDIRECTTARGET pTgt, CONT
 
     ////////////////////////////////////////////////////
     // Now redirect the thread to the helper function
-
```

Comment:
The only change to this file is whitespace. Can you revert this change so `git` / GitHub don't see it as a change?

Reply:
I removed it but let me see if I can revert it